org-gtd-capture.el

```diff
@@ -37,7 +37,6 @@ Wraps the function `org-capture' to ensure the inbox exists.
 For GOTO and KEYS, see `org-capture' documentation for the variables of the same name."
   (interactive)
   (with-org-gtd-context
-      (org-gtd--inbox-file)
```

Comment:
It's not needed anymore here, since the target function in the capture template takes care of it.

Reply:
This is, however, a problem if the user changes the org-capture templates, and if we are going to give the user more power in the capture templates, then this code can break because of user actions, so I would rather not leave that to chance.